### ISSUE TITLE
The server doesn't work correctly with paths that start with a number

### ISSUE THREAD

**Issue by stancl** [tightenco/jigsaw-docs-template#32]
I'm working on an implementation of #31 but I found that if you have `source/1.x`, `source/2.x` etc instead of `source/docs`, the server will return a 404, but if you append `/index.html` it works.

**Comment by stancl** [comment]
Sorry, this should probably go to tightenco/jigsaw. Should I close this and open the issue there?

**Comment by damiani** [comment]
Yes, if this is a bug with Jigsaw parsing directories, please open an issue there.

**Comment by stancl** [comment]
It's not an issue with Jigsaw itself, it's an issue with the live server.
The same content works correctly on Netlify.

Dne st 18. zář 2019 14:18 uživatel Keith Damiani <notifications@github.com>
napsal:

**Comment by stancl** [comment]
Could you perhaps transfer this issue to tightenco/jigsaw?
